crates/tests-e2e/tests/close_position.rs

```diff
@@ -24,4 +25,24 @@ async fn can_collab_close_position() {
     wait_until!(test.app.rx.position().unwrap().position_state == PositionState::Closing);
 
     // TODO: Assert that the position is closed in the app and the coordinator
+
+    tracing::info!("Position closed");
+
+    // Ensure we sync the wallet info after funding
+    spawn_blocking(move || api::refresh_wallet_info().expect("to succeed"))
+        .await
+        .unwrap();
+
+    let balance_at_closing = test.app.rx.wallet_info().unwrap().balances.on_chain;
+
+    let invoice =
+        spawn_blocking(move || api::create_invoice_with_amount(50_000).expect("to succeed"))
+            .await
+            .unwrap();
+    api::decode_invoice(invoice.clone()).expect("to decode invoice we created");
+
+    let client = reqwest::Client::new();
+    pay_with_faucet(&client, invoice).await.unwrap();
+
+    wait_until!(test.app.rx.wallet_info().unwrap().balances.on_chain > balance_at_closing);
```

Comment:
Unless I'm missing something, you are not going on-chain here. So this will always fail.

Reply:
it would, indeed, but I never got to that spot due to the `no_route` error beforehand.